src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -18,7 +18,7 @@
         </span>
 
         <!-- Firm info tooltip -->
-        <v-tooltip v-if="isChangeRegFiling || isFirmConversionFiling"
+        <v-tooltip v-if="isChangeRegFiling || isFirmConversionFiling || isFirmCorrectionFiling"
```

Comment:
Would it make more sense to simply use the "is firm" getter here?

Reply:
I think so too: `isEntityTypeFirm`.

Reply:
LMK if you're not going to change this.

Reply:
It's now `isEntityTypeFirm`.